src/hello-world/hello-world.component.html

```diff
@@ -0,0 +1,3 @@
+<div>Hello {{ name }}</div>
+<input type="text" [value]="name" (input)="name = $event.target.value"/>
```

Comment:
we need an actual example app. I'll ask in the GDE channel to see if we can get something good